devilizer/__main__.py

```diff
@@ -0,0 +1,32 @@
+import random
+import time
+
+from rich.console import Console
+from rich.live import Live
+from rich.panel import Panel
+from rich.align import Align
+
+from devilizer.sorting.bubble_sort import bubble_sort
+from devilizer.visualizer.display_items import display_items
+
+
+def generate_table(display: str) -> Panel:
+    """Gnerate a panel to display using the rich library.
+
+    Args:
+        display (str): The visualized values to display.
+
+    Returns:
+        rich.Panel: The generated a panel.
+    """
+    table = Panel(display, expand=False, title="[bold blue]Devilizer[/bold blue]")
+    return Align(table, align="center")
+
+
+console = Console()
+lst = random.sample(list(range(1, 20)), 19)
```

Comment:
Range is a sequence also, so the list is unnecessary.
```suggestion
lst = random.sample(range(1, 20), 19)
```

Reply:
Does this work? I thought that `random.sample` required a list.

Reply:
It needs a sequence, so it can access values by index.